### ISSUE TITLE
Can't run Linux Firefox on WSL

### ISSUE THREAD

**Issue by tstackhouse** [karma-runner/karma-firefox-launcher#107]
There's no way to override the changes made in #96, I can set `FIREFOX_BIN` to use the linux firefox, but because of the wsl detection it tries to convert the profile path and causes the linux wires to get crossed.

**Comment by johnjbarton** [comment]
What is a WSL?

**Comment by tstackhouse** [comment]
> 
> 
> What is a WSL?

Windows Subsystem for Linux.  the referenced PR added a check for that platform and overrides the behaviour of the launcher to use the windows version of Firefox without even checking if a Linux version is available.

**Comment by birtles** [comment]
Yeah, sorry about that. I didn't think anyone was actually trying to running Linux Firefox from WSL. It seems the majority of people are trying to run Windows Firefox but we should allow both.

**Comment by tstackhouse** [comment]
No worries! I forked the repo to try to submit a PR, but I'm not familiar enough with how karma launchers work in order to make it a configurable option, or at least with how the check is currently implemented, as it seems like it happens at a global level (like the ternary in the launch commands)  The naive way to implement it would be to simply check for the existence of firefox inside WSL before deferring to the windows version.

**Comment by tstackhouse** [comment]
Just wanted to follow up on this, I've got a potential fix that will work to bypass `isWsl` should firefox be installed in WSL.  That said, I've noticed a caveat: 

* Firefox versions greater than 59 (probably more like 67/68) don't work, I know the most recent version (72 at the time of writing) doesn't.  Running the command manually gives an "Exiting due to channel error." error, from what I can tell it's related to multi-process and DBus/IPC.  Basically wonky low-level WSL stuff that will supposedly be fixed in WSL2.

I've been down this path mainly because running the windows Firefox, at least in the way my system is configured, where I'm also using Firefox as my primary browser, communication between Karma on WSL and Firefox on windows is janky at best, with sessions disconnecting, and Firefox spamming me with "XUL Runner Error" dialogs.  I may need to spend some time investigating further to see why it fails like it does.
